### PR TITLE
feat: Add radio button component

### PR DESCRIPTION
**Pull request by GoOz**
Added `<Radio />`component.

Preview: https://gooz.github.io/cozy-ui/react/#radio

### CODE REVIEW

File: react/Radio/index.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  value: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  error: PropTypes.bool,
+  label: PropTypes.string.isRequired
+}
+
+Radio.defaultProps = {
+  className: '',
+  value: '',
+  name: '',
```
Comment (ptbrowne) [inline react/Radio/index.jsx]: It is unnecessary to specify all those '', and it is breaking the isRequired mechanism. An empty string will pass the isRequired test and I don't think that is what you want.

**Comment by ptbrowne** [comment]
Otherwise, good style, and nice effect :)

File: react/Radio/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Radio.propTypes = {
+  className: PropTypes.string,
+  value: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  error: PropTypes.bool,
+  label: PropTypes.string.isRequired
+}
+
+Radio.defaultProps = {
+  className: '',
```
Comment (ptbrowne) [inline react/Radio/index.jsx]: It is unnecessary to put this ''

**Comment by GoOz** [comment]
[Add Input Radio component](https://trello.com/c/AzWaVK7n/127-add-input-radio-component)

**Pull request merged**